labelbox/data/generator.py

```diff
@@ -33,7 +33,15 @@ class PrefetchGenerator:
     Useful for modifying the generator results based on data from a network
     """
 
-    def __init__(self, data: Iterable[Any], prefetch_limit=20, num_executors=4):
+    #maybe change num exec to just 1, and if 1, make sync
+    #instead of self.get qeue in next, itll return just self._data.next
+    #kwarg on export for multithread, and all other things that use prefetch
+
+    def __init__(self,
+                 data: Iterable[Any],
+                 prefetch_limit=20,
+                 num_executors=4,
+                 multithread: bool = False):
```

Comment:
Instead of a multithread param, lets just say when num_executors == 1 we don't use threading.